ios/Classes/SITFLNativeMapView.swift

```diff
@@ -256,4 +265,31 @@ extension SITFLNativeMapView : OnMapReadyListener, OnPoiSelectionListener, OnNav
         }
     }
     
```

Comment:
Podería chegar a ser necesario algun erro na creación do Poi, ou que o usuario non creou finalmente ningun poi a pesar de entrar non modo de creación de customPoi?

Reply:
Ahora mismo no lo estamos contemplando, pero quizás en el futuro se podría añadir un nuevo listener para la cancelación del custom poi, o error en caso de que sea necesario. Si creéis que es importante podemos mirar de meterlo ahora